From 1cb15f248d8aff75b33316966e4e0c443fa8fe02 Mon Sep 17 00:00:00 2001
From: Charley Robinson <charley@tokbox.com>
Date: Thu, 5 Jun 2014 13:42:08 -0600
Subject: [PATCH] MINOR Cleanup Hello-World view controller

* Keep working-copy credentials out of the repository.
* Remove optional callbacks. This app is meant to demonstrate minimum
  functionality.
---
 1.Hello-World/Hello-World/ViewController.m | 13 +++----------
 1.Hello-World/OpenTok.framework            |  2 +-
 2 files changed, 4 insertions(+), 11 deletions(-)

diff --git a/1.Hello-World/Hello-World/ViewController.m b/1.Hello-World/Hello-World/ViewController.m
index 4eba79b..4f9b4d1 100644
--- a/1.Hello-World/Hello-World/ViewController.m
+++ b/1.Hello-World/Hello-World/ViewController.m
@@ -24,11 +24,11 @@ static double widgetWidth = 320;
 // *** Fill the following variables using your own Project info  ***
 // ***          https://dashboard.tokbox.com/projects            ***
 // Replace with your OpenTok API key
-static NSString* const kApiKey = @"100";
+static NSString* const kApiKey = @"";
 // Replace with your generated session ID
-static NSString* const kSessionId = @"2_MX4xMDB-flR1ZSBOb3YgMTkgMTE6MDk6NTggUFNUIDIwMTN-MC4zNzQxNzIxNX4";
+static NSString* const kSessionId = @"";
 // Replace with your generated token
-static NSString* const kToken = @"T1==cGFydG5lcl9pZD0xMDAmc2RrX3ZlcnNpb249dGJwaHAtdjAuOTEuMjAxMS0wNy0wNSZzaWc9OTVjMmUxYmNkNGI0NDNiM2UyNDA0N2Y0NDk0ZGQ1MWUyZWMzZjUzNzpzZXNzaW9uX2lkPTJfTVg0eE1EQi1mbFIxWlNCT2IzWWdNVGtnTVRFNk1EazZOVGdnVUZOVUlESXdNVE4tTUM0ek56UXhOekl4Tlg0JmNyZWF0ZV90aW1lPTEzOTg5MDcxMzQmcm9sZT1tb2RlcmF0b3Imbm9uY2U9MTM5ODkwNzEzNC44MDcxNzMyNDIzNTEwJmV4cGlyZV90aW1lPTE0MDE0OTkxMzQ=";
+static NSString* const kToken = @"";
 
 // Change to NO to subscribe to streams other than your own.
 static bool subscribeToSelf = NO;
@@ -264,11 +264,4 @@ didFailWithError:(OTError*)error
     });
 }
 
-- (void)   session:(OTSession*)session
-receivedSignalType:(NSString*)type
-    fromConnection:(OTConnection*)connection
-        withString:(NSString*)string
-{
-    NSLog(@"receivedSignalType %@", string);
-}
 @end
\ No newline at end of file
1.8.5.2 (Apple Git-48)

